woocommerce/woocommerce
# Show marketplace suggestions in orders list empty state

### All Submissions:




### Changes proposed in this Pull Request:

Adds support for orders list suggestions.

### How to test the changes in this Pull Request:

1. Checkout this branch and set it up as the WooCommerce plugin in your development environment.
2. Log in as an admin, or a user with `install_plugins` permission.
3. View empty orders list. Note that this requires no orders – suggestions are only displayed in the empty state.
4. See some useful info about different options for building out or scaling up your store.

### Other information:


This PR currently contains hard-coded test data – this will be removed when the latest data has been uploaded to S3. cc @mattyza @kovshenin 

### Screenshot 

Note the content (header, footer, titles, plugins, button text etc) is all test data; the final copy is still being prepared.

<img width="1347" alt="screen shot 2019-02-22 at 4 18 23 pm" src="https://user-images.githubusercontent.com/4167300/53217807-68b6b000-36be-11e9-95e6-7339899d518f.png">
